src/plots/cartesian/axes.js

```diff
@@ -1805,7 +1805,7 @@ axes.drawTicks = function(gd, ax, opts) {
     ticks.enter().append('path')
         .classed(cls, 1)
         .classed('ticks', 1)
-        .classed('crisp', 1)
+        .classed('crisp', !opts.noCrisp)
```

Comment:
Yeah, that's cool! But I'm not a huge fan of negated option names - could we use:
`opts.crisp !== false`?

Reply:
Done in https://github.com/plotly/plotly.js/pull/3254/commits/016b8e485ba5e586e4b4dbfbf77c988d2298f089